### ISSUE TITLE
Error: object 'model_par' not found in WaveletLSTM function

### ISSUE THREAD

**Issue by nikshu35** [cran/WaveletLSTM#1]
“I tried to run the example code from the package, but I get this error:”
Error in WaveletLSTM(ts = y) : object 'model_par' not found
R version 4.4.1 (2024-06-14 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26100)

Matrix products: default

locale:
[1] LC_COLLATE=English_India.utf8  LC_CTYPE=English_India.utf8   
[3] LC_MONETARY=English_India.utf8 LC_NUMERIC=C                  
[5] LC_TIME=English_India.utf8    

time zone: Asia/Calcutta
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] readxl_1.4.3      WaveletLSTM_0.1.0

loaded via a namespace (and not attached):
  [1] pROC_1.18.5          rlang_1.1.4          magrittr_2.0.3      
  [4] tseries_0.10-58      TSLSTM_0.1.0         compiler_4.4.1      
  [7] png_0.1-8            vctrs_0.6.5          reshape2_1.4.4      
 [10] quadprog_1.5-8       stringr_1.5.1        pkgconfig_2.0.3     
 [13] utf8_1.2.4           prodlim_2024.06.25   pracma_2.4.4        
 [16] nloptr_2.1.1         purrr_1.0.2          jsonlite_1.8.8      
 [19] recipes_1.1.0        keras_2.15.0         caretForecast_0.1.1 
 [22] parallel_4.4.1       tensorflow_2.16.0    R6_2.5.1            
 [25] stringi_1.8.4        RColorBrewer_1.1-3   reticulate_1.42.0   
 [28] wavelets_0.3-0.2     parallelly_1.37.1    rpart_4.1.23        
 [31] lmtest_0.9-40        lubridate_1.9.3      cellranger_1.1.0    
 [34] Rcpp_1.0.12          iterators_1.0.14     smooth_4.2.0        
 [37] future.apply_1.11.2  zoo_1.8-12           base64enc_0.1-3     
 [40] MAPA_2.0.7           Matrix_1.7-0         splines_4.4.1       
 [43] nnet_7.3-19          timechange_0.3.0     tidyselect_1.2.1    
 [46] rstudioapi_0.16.0    timeDate_4041.110    codetools_0.2-20    
 [49] curl_5.2.1           listenv_0.9.1        lattice_0.22-6      
 [52] tibble_3.2.1         plyr_1.8.9           quantmod_0.4.26     
 [55] withr_3.0.0          urca_1.3-4           future_1.33.2       
 [58] survival_3.6-4       xts_0.14.0           texreg_1.39.4       
 [61] pillar_1.9.0         whisker_0.4.1        foreach_1.5.2       
 [64] stats4_4.4.1         generics_0.1.3       TTR_0.24.4          
 [67] rprojroot_2.0.4      forecast_8.23.0      ggplot2_3.5.1       
 [70] munsell_0.5.1        scales_1.3.0         greybox_2.0.4       
 [73] globals_0.16.3       xtable_1.8-4         class_7.3-22        
 [76] glue_1.7.0           tools_4.4.1          data.table_1.15.4   
 [79] ModelMetrics_1.2.2.2 gower_1.0.1          grid_4.4.1          
 [82] tsutils_0.9.4        ipred_0.9-14         colorspace_2.1-0    
 [85] nlme_3.1-164         fracdiff_1.5-3       cli_3.6.3           
 [88] tfruns_1.5.3         fansi_1.0.6          lava_1.8.0          
 [91] dplyr_1.1.4          gtable_0.3.5         zeallot_0.1.0       
 [94] digest_0.6.36        caret_7.0-1          lifecycle_1.0.4     
 [97] hardhat_1.4.0        httr_1.4.7           here_1.0.1          
[100] statmod_1.5.0        MASS_7.3-61 
packageVersion("waveletlstm")
[1] ‘0.1.0’
Model: "sequential_1"
____________________________________________________________________________________
 Layer (type)                        Output Shape                      Param #      
====================================================================================
 lstm_1 (LSTM)                       (None, 1, 20)                     2560         
 dense_1 (Dense)                     (None, 1, 1)                      21           
====================================================================================
Total params: 2,581
Trainable params: 2,581
Non-trainable params: 0
____________________________________________________________________________________
Epoch 1/25
70/70 [==============================] - 2s 10ms/step - loss: 0.1039 - mae: 0.2583
Epoch 2/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0445 - mae: 0.1692
Epoch 3/25
70/70 [==============================] - 0s 4ms/step - loss: 0.0425 - mae: 0.1620
Epoch 4/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0423 - mae: 0.1684
Epoch 5/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0422 - mae: 0.1644
Epoch 6/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0399 - mae: 0.1562
Epoch 7/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0389 - mae: 0.1519
Epoch 8/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0396 - mae: 0.1595
Epoch 9/25
70/70 [==============================] - 0s 4ms/step - loss: 0.0388 - mae: 0.1560
Epoch 10/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0379 - mae: 0.1504
Epoch 11/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0376 - mae: 0.1523
Epoch 12/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0365 - mae: 0.1473
Epoch 13/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0367 - mae: 0.1459
Epoch 14/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0357 - mae: 0.1431
Epoch 15/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0357 - mae: 0.1413
Epoch 16/25
70/70 [==============================] - 0s 4ms/step - loss: 0.0353 - mae: 0.1400
Epoch 17/25
70/70 [==============================] - 0s 4ms/step - loss: 0.0352 - mae: 0.1413
Epoch 18/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0347 - mae: 0.1394
Epoch 19/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0343 - mae: 0.1383
Epoch 20/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0345 - mae: 0.1372
Epoch 21/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0337 - mae: 0.1372
Epoch 22/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0337 - mae: 0.1361
Epoch 23/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0330 - mae: 0.1322
Epoch 24/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0332 - mae: 0.1358
Epoch 25/25
70/70 [==============================] - 0s 5ms/step - loss: 0.0324 - mae: 0.1306
70/70 [==============================] - 1s 2ms/step - loss: 0.0323 - mae: 0.1362
70/70 [==============================] - 0s 1ms/step
17/17 [==============================] - 0s 2ms/step - loss: 0.0343 - mae: 0.1606
17/17 [==============================] - 0s 2ms/step
Error in WaveletLSTM(ts = y) : object 'model_par' not found

**Comment by gaborcsardi** [comment]
Hi, this is a read only mirror of CRAN, please see the package authors in the `DESCRIPTION` file. Look for `Maintainer`, `BugReports` and `URL`. Thanks!
